backend/src/zango/cli/start_project.py

```diff
@@ -228,19 +228,28 @@ def start_project(
     # Creating Public Tenant
     create_public_tenant(platform_domain_url=platform_domain_url)
 
-    # Prompting default platform user details
-    if not platform_username:
-        click.echo("Please enter platform user email")
-        platform_username = click.prompt("Email")
-    if not platform_user_password:
-        platform_user_password = click.prompt(
-            "Password", hide_input=True, confirmation_prompt=True
+     # Prompting default platform user details
+    while True:
+        platform_username = None
+        platform_user_password = None
```

Comment:
We can shift this code
```python
platform_username = None
platform_user_password = None 
```
from here to the `else` block (line no. 249), where we check if user creation has failed. This will work in both cases: if the platform username and password are supplied via argument or from a prompt. Essentially, we will not initially set them to None. Instead, if user creation fails, we will set them to None so that new values can be provided by the user.

Reply:
ahh ok that makes sense thanks for the explanation Ill make the change